format/spec.md

```diff
@@ -654,6 +656,7 @@ A snapshot consists of the following fields:
 | _optional_ | _required_ | _required_ | **`summary`**                | A string map that summarizes the snapshot changes, including `operation` (see below)                                               |
 | _optional_ | _optional_ | _optional_ | **`schema-id`**              | ID of the table's current schema when the snapshot was created                                                                     |
 |            |            | _optional_ | **`first-row-id`**           | The first `_row_id` assigned to the first row in the first data file in the first manifest, see [Row Lineage](#row-lineage) |
+|            |            | _optional  | **`added-rows`**         | The number of newly added rows in this snapshot. Required if [Row Lineage](#row-lineage) is enabled | 
```

Comment:
format nit: 
- missing trailing underscore from `_optional`
- 5th and 6th `|` seem to be misaligned from rows above

Reply:
Normally we don't do the vertical alignment, but it's pretty easy here so i'll fix it